json-lint/utils/schemaUtils.js

```diff
@@ -11,13 +11,27 @@
  * is strictly forbidden unless prior written permission is obtained
  * from Marfeel Solutions SL.
  */
+const JSON_MERGE_COMMAND = 'mrf-json';
 
 const path = require('path');
 const fs = require('fs');
+const spawnSync = require('child_process').spawnSync;
 
 const isJson = (filePath) => filePath.endsWith('.json');
 const isNotPrivate = (fileName) => fileName[0] !== '.';
 
+function getMarfeelExtendedJson(jsonPath) {
+	const ls = spawnSync(JSON_MERGE_COMMAND , [ jsonPath ]);
+	try{
+		return JSON.parse(ls.stdout)
```

Comment:
I'm not sure how mrf-json works, but it should always return valid json on the stdout or return an error on the stderr, right?
Couldn't you check the ls.status or ls.error instead of trying to parse the stdout? I think it's a more declarative and solid check